### ISSUE TITLE
Visualizing with Tensorboard

### ISSUE THREAD

**Issue by LK1711** [smallcorgi/Faster-RCNN_TF#280]
Has anyone integrated Tensorboard while training/testing their own dataset? I do not understand which file the line of code mentioned here should go:[https://www.tensorflow.org/programmers_guide/summaries_and_tensorboard](url)

Could someone please help! 

Thanks in advance!

**Comment by walsvid** [comment]
I have just completed the work of integrating tensorboard into faster-rcnn. The code is base on this repo. For more details you can reference my repo: Visit https://github.com/walsvid/Faster-RCNN-TensorFlow

The added code is mainly summary writer and several scalar summary of loss.

If you want to know the specific implementation details, you can view these commits: walsvid/Faster-RCNN-TensorFlow@82a072a2e07ec1a78eddd08412b0e883584966e0, walsvid/Faster-RCNN-TensorFlow@0f606601995f505e94f80173fb411e09dfe1add5 and walsvid/Faster-RCNN-TensorFlow@7ba482eb836f679d90345b34da1bb70c429ec7d1.

**Comment by meijie0401** [comment]
@walsvid Hi thanks for your code for tensorboard. But I also want to see the loss on validation dataset. Do you know how to feed validation dataset to feed_dict ? thanks a lot!!!!!

**Comment by walsvid** [comment]
@meijie0401 Since the dataset of Faster-rcnn is `trainval` and `test`, the `roidb` used in the current code does not distinguish between train and val, so you can refer to this part of the code to build your model with `val_roidb`: [Reference](https://github.com/endernewton/tf-faster-rcnn/blob/a3279943cbe6b880be34b53329a4fe3f971c2c37/tools/trainval_net.py#L137)

**Comment by meijie0401** [comment]
@walsvid Thank you!! another quick questoin: is there normalization for input image in this repo? I can't l locate normolization in this code.
